hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
+  tag: string
+  data?: Data
+  children: VElement<{}>[]
+}
+
+export type VElement<Data> = VNode<Data> | string
```

Comment:
Now it is used as child, but I dont think it is good to limit it at this role by naming.
I do really think `VirtualElement` is good. @pspeter3,did an awesome job there.